compass/Coordinator.java

```diff
@@ -113,6 +114,11 @@ private void shutdownHook() {
   }
 
   public static void main(String[] args) throws Exception {
+
+    // Limit DNS caching for resolved and failed records to 5 seconds
+    Security.setProperty("networkaddress.cache.ttl" , "5");
+    Security.setProperty("networkaddress.cache.negative.ttl" , "5");
```

Comment:
Do we want to have it hardcoded? This means that if we want to change the value we need to release a new compass.
Maybe we can pass it as `-D` config when running through the cli? Or add it to compass config file?
@geminoz you have a preference?

Reply:
Yeah, I thought about the `-D` too. But I am not really positive about that: we should not keep this optional and prone to be forgotten.

Reply:
We can check whether the value is the default, and if it is, force it to our default.
This way we get the best of both worlds :-)

Reply:
Too convoluted. There is no actual reason why a user wants to really change this. No big deal a DNS resolve on each milestone tick.